react/features/base/util/interceptComponent.js

```diff
@@ -26,6 +32,17 @@ const _RULES = [
         if (OS === 'android' || OS === 'ios') {
             return UnsupportedMobileBrowser;
         }
+    },
+    () => {
+        if (APP.unsupportedBrowser) {
+            const { isOldBrowser } = APP.unsupportedBrowser;
+
+            if (isOldBrowser) {
+                return UnsupportedDesktopBrowser;
```

Comment:
If I follow the logic of the isOldBrowser property to lib-jitsi-meet, this would mean that someone on an old FF version will see the message: "Please try again with Chrome, Firefox, ...". Am I right? If this is the case we should probably change the message to "Please try again with the latest version Chrome, Firefox..." or something in those lines.